src/main/java/mate/academy/spring/controller/OrderController.java

```diff
@@ -40,8 +41,10 @@ public OrderResponseDto completeOrder(@RequestParam Long userId) {
     }
```

Comment:
In the line 38, according to task, you should "Replace all ?userId @RequestParam in your code with Authentication object"